### DISCUSSION TITLE
Welcome to Diffractor Discussions!

### DISCUSSION THREAD

**Discussion by ZacWalk** [diffractor/diffractor#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aceweo** [comment]
Hi, How do I get Diffractor to "open with" Google Earth app which is installed on my PC?

Thanks
Tony

**Reply by ZacWalk** [reply]
You mean opening the location of a photo in gearth? I will need to do some testing on that but I am away on holiday for a few weeks. Try when I get back.

**Comment by originalnightshift** [comment]
hi, I liked the program. Keep the Russian translation 
[ru..zip](https://github.com/user-attachments/files/29439669/ru.zip)

**Reply by ZacWalk** [reply]
Thanks - much appreciated. I will add in one of the next releases. Zac

**Comment by ZacWalk** [comment]
Thanks. Russian is added in the latest releasee.
